src/formpack/utils/replace_aliases.py

```diff
@@ -66,25 +66,37 @@ def aliases_to_ordered_dict(_d):
     'geopoint': ['gps'],
 })
 
-selects = aliases_to_ordered_dict({
-    'select_multiple': [
-        'select all that apply',
-        'select multiple',
-        'select many',
-        'select_many',
-        'select all that apply from',
-        'add select multiple prompt using',
-    ],
-    'select_one_external': [
-        'select one external',
-    ],
-    'select_one': [
-        'select one',
-        'select one from',
-        'add select one prompt using',
-        'select1',
-    ],
-})
+# keys used in `_expand_type_to_dict()` to handle choices argument
+selects = aliases_to_ordered_dict(
+    {
+        'select_multiple': [
+            'select all that apply',
+            'select multiple',
+            'select many',
+            'select_many',
+            'select all that apply from',
+            'add select multiple prompt using',
+        ],
+        'select_multiple_from_file': [
+            'select multiple from file',
+        ],
+        'select_one_external': [
+            'select one external',
+        ],
+        'select_one': [
+            'select one',
+            'select one from',
+            'add select one prompt using',
+            'select1',
+        ],
+        'select_one_from_file': [
+            'select one from file',
+        ],
+        'rank': [
+            'rank',
+        ],
```

Comment:
i'd suggest an empty list since `rank` has no aliases. `aliases_to_ordered_dict()` handles this well:
```
In [3]: aliases_to_ordered_dict({'rank': []})                                                                                        
Out[3]: OrderedDict([('rank', 'rank')])
```